spec/oppiabotSpec.js

```diff
@@ -102,24 +119,18 @@ describe('Oppiabot\'s', () => {
       expect(apiForSheetsModule.checkClaStatus.calls.count()).toEqual(1);
     });
 
-    it('should be called with three arguments for the given payload', () => {
+    it('should be called with one argument for the given payload', () => {
       expect(
-        apiForSheetsModule.checkClaStatus.calls.argsFor(0).length).toEqual(3);
+        apiForSheetsModule.checkClaStatus.calls.argsFor(0).length).toEqual(1);
     });
 
     it('should be called with the correct username for the given payload',
       () => {
-        expect(
-          apiForSheetsModule.checkClaStatus.calls.argsFor(0)[0])
-          .toEqual('testuser7777');
+        const context = apiForSheetsModule.checkClaStatus.calls.argsFor(0)[0];
+        expect(context.payload.pull_request.user.login).toEqual(
+          'testuser7777');
       });
 
-    it('should be called for a pull request for the given payload', () => {
-      expect(
-        apiForSheetsModule.checkClaStatus.calls.argsFor(0)[2])
-        .toEqual(true);
-    });
-
```

Comment:
Why is this removed?

Reply:
I'm not sure what was being tested here because `apiForSheetsModule` expects just one argument, and the check is testing for the third argument.

Reply:
I'm not sure what we should test here.

Reply:
I think the test is related to contextData, can you try to figure out what was being compared with true in the original test.

Reply:
@ankita240796 I think it checks that the data gotten is a pull request data
https://github.com/oppia/oppiabot/blob/3aa31ef80b5af0789e34d608ef85ec9a11becd61/index.js#L32
https://github.com/oppia/oppiabot/blob/3aa31ef80b5af0789e34d608ef85ec9a11becd61/lib/apiForSheets.js#L114

Reply:
So, we should test the same in the current tests as well.

Reply:
I don't think it is necessary to check if it is a pull request data or not since we are only calling the `apiForSheetsModule.checkClaStatus` function when a pull request is opened.
I believe this was done initially because the checkClaStatus was also called when an issue comment was created.
https://github.com/oppia/oppiabot/blob/511f4f06332f73b2efa9a258305d1821d3aaacac/index.js#L15

Reply:
Ok, sg.